src/main/liberty/config/server.env

```diff
@@ -0,0 +1 @@
+key=7JS0-6XPE-OSNL-QR6X
```

Comment:
to ensure keys never get checked in like this, lets create a `.gitignore` in this dir that ignores the server.env file